polkadot/node/subsystem-bench/src/cli/subsystem-bench.rs

```diff
@@ -120,19 +123,25 @@ impl BenchCli {
 			None
 		};
 
-		let test_sequence = TestSequence::new_from_file(Path::new(&self.path))
-			.expect("File exists")
-			.test_configurations;
+		let test_sequence: Vec<CliTestConfiguration> =
+			TestSequence::new_from_file(Path::new(&self.path))
+				.expect("File exists")
+				.test_configurations
+				.into_iter()
+				.map(|CliTestConfiguration { objective, test_config }| CliTestConfiguration {
```

Comment:
It looks a bit complicated.

Reply:
Would it be good for me to change CliTestConfiguration struct from using TestConfiguration to TestConfigurationBuilder ? This would have it deserialize the yaml file in TestConfigurationBuilder and build to generate TestConfiguration. @AndreiEres

Reply:
I'm not sure but we can try